ipam/client/backends/phpipam.py

```diff
@@ -102,6 +99,12 @@ def get_next_free_ip(self, subnet):
         subnetips = subnet.iter_hosts()
         # Get allocated ip addresses from database
         usedips = self.get_allocated_ips_by_subnet_id(subnetid)
+
+        # Dirty hack, as netaddr has a bug with /127 subnets
```

Comment:
what is this bug for /127 subnets?

Reply:
As discussed: https://github.com/drkjam/netaddr/pull/168
Comment added.